homeyess/templates/dashboard/company.html

```diff
@@ -101,7 +101,7 @@ <h2>Jobs Posted</h2>
 				</tr>
 				{% for job in job_posts %}
 			   		<tr>
-						<td style="padding:15px"><a href="#">Open</a></td>
+						<td style="padding:15px"><a href="{% url 'edit_job' user.profile.pk job.id %}">Open</a></td>
```

Comment:
I do not think you need the `user.profile.pk` here because the user is already logged in.